TAC/Meetings/2023/2023-09-07/CCC_TAC_Minutes_2023-09-07.md

```diff
@@ -0,0 +1,154 @@
+Welcome Kanth Ghatraju (Oracle).
+Welcome Kevin McCarthy (Inpher).
+
+**Meeting Minute Approval**
+
+Proposed: That the minutes of the August 23, 2023 meeting of the Technical Advisory Council of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.
+
+
+**Cross-Project goal updates**
+
+* Lily Sturmann (LS) gave an update on cross-project goals. LS and Eric Voit put together a survey and sent it to the project mentors. There has been 1 response so far. 
+* LS will reach out to Ben on an action item that Kurt marked as done.
+
+
+**Tech Talk: MPC/FL + TEEs; Kevin McCarthy (Inpher)**
+
+* Kevin McCarthy (KM) presented his talk on MPC/FL + TEE and FHE.
+* Inpher specializes in the cryptographic space.
+* Organizations are asking for hybrid PET solutions.
+* Attention to GRC SIG: https://iabtechlab.com/working-groups/privacy-enhancing-technologies-working-group/
+* Question: Why is CC not enough? Technology lives with what clients already use: Azure, AWS, etc. Inpher works with their clients strategy.
+* Kevin McCarthy will put together a list of regulators to send to Mike Bursell.
+* Comment: CC should always be included with other PETs.
+* CC hasn't penetrated the market yet.
+* Dave Thaler (DT) brought up that Manu posted a link in chat (https://iabtechlab.com/wp-content/uploads/2023/02/FINAL-DRAFT-PUBLIC-COMMENT-Open-Private-Join-Activation-IAB-Tech-Lab.pdf) where the CCC would disagree with the definition of TEE. DT asked KM if IAB would be willing to update the TEE definition should the CCC contact them. KB believes the IAB will update the definition. 
+
+***Action item: Kevin McCarthy will connect IAB with Mike Bursell to request to update the their definition of TEE.***
+
+
+**Rename stages per agreement in 2023-06-15 TAC meeting**
+
+* https://github.com/confidential-computing/governance/pull/183
+* DT made a proposal to include a growth plan at the first stage to get to graduation stage.
+* The group agrees that this makes sense.
+* MB asked if the GB should vote on the charter. DT: The TAC Chair should report it to the GB for visibility.
+
+*** Action item for all project mentors: Update the stage for each project in the project readme***
+
+
+**Project Proposal: Rust SPDM**
+
+* Samuel Ortiz (Rivos) is the new maintainer.
+* https://github.com/intel/rust-spdm
+* Want to know how this project promotes confidential computing.
+
+
+**web page updates; mentors**
+
+* DT raised a concern on whether or not the diagram deters project submissions. Suggested to move to the current projects landing page.
+* MB will work on getting a web content team together within the Outreach committee.
+
+
+**CISA RFI**
+
+* The response is due on October 9, 2023.
+* RFI response will be sent to the board a week prior to the due date to review.
+* Working doc: https://docs.google.com/document/d/1hEZZfsmMtBpOYkCpRTMNYFbY9_qEYK_VxMRlXhQxfEU/edit#heading=h.3pkkmtscvc8q
+
```

Comment:
```suggestion
* TAC contributors are encouraged to get bulleted concepts into the google doc immediately. MB will be creating a first draft imminently and it will be easier to incorporate concepts at the outset rather than through revisions.
```